script/deploy/facets/DeployLiFiTimelockController.s.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { DeployScriptBase } from "./utils/DeployScriptBase.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { LiFiTimelockController } from "lifi/Security/LiFiTimelockController.sol";
+
+contract DeployScript is DeployScriptBase {
+    using stdJson for string;
+
+    constructor() DeployScriptBase("LiFiTimelockController") {}
+
+    function run()
+        public
+        returns (LiFiTimelockController deployed, bytes memory constructorArgs)
+    {
+        constructorArgs = getConstructorArgs();
+
+        deployed = LiFiTimelockController(
+            deploy(type(LiFiTimelockController).creationCode)
+        );
+    }
+
+    function getConstructorArgs() internal override returns (bytes memory) {
+        string memory path = string.concat(
+            root,
+            "/deployments/",
+            network,
+            ".",
+            fileSuffix,
+            "json"
+        );
+
+        address diamondAddress = _getConfigContractAddress(
+            path,
+            ".LiFiDiamond"
+        );
+
+        // get minDelay from config file
+        string memory timelockConfigPath = string.concat(
+            root,
+            "/config/timelockController.json"
```

Comment:
`timelockController.json` but the config file is named `timelockcontroller.json`